### ISSUE TITLE
Errors running resolver tests

### ISSUE THREAD

**Issue by fernandreu** [kornilova203/matlab-IntelliJ-plugin#49]
Hi, I was having a look at the resolver tests, but many of them were giving me errors of two types. 
I would say both errors occur because I am using a different platform (Windows 10) / configuration. I have found a workaround for both, but there are probably better ways of doing it.

In `testFunctionInBlockUnresolved`, `testFunctionUnresolved`, `testVariableUnresolved` and `testVarInsideFunctionUnresolved`, I obtain:

    java.lang.AssertionError: File accessed outside allowed roots: file://C:/matlab-IntelliJ-plugin/testData/resolve/NameOfTheMatlabFile.m;
    Allowed roots: [a list of paths none of which is the one above]

Reading a bit about the error, I found a way to disable that check by setting the environment variable `NO_FS_ROOTS_ACCESS_CHECK` to `1`.

The other error occurs in many other tests except a few:

    kotlin.TypeCastException: null cannot be cast to non-null type com.github.korniloval.matlab.MatlabReference

        at com.github.korniloval.matlab.resolve.ResolveTest.getExpectedDeclaration(ResolveTest.kt:76)
        at com.github.korniloval.matlab.resolve.ResolveTest.getExpectedDeclaration(ResolveTest.kt:56)
        at com.github.korniloval.matlab.resolve.ResolveTest.doTest(ResolveTest.kt:36)
        at com.github.korniloval.matlab.resolve.ResolveTest.testVariableInCatchBlock(ResolveTest.kt:33)
        
Basically, it cannot find a reference at refOffset. It seems it is because my repo was checked out with Windows line endings (CRLF). If I use LF or CR instead, they all pass.

**Comment by kornilova203** [comment]
@fernandreu,
Sorry for very-very slow reply
I have never had a `File accessed outside allowed roots` error. Don't know why it may occur. 99% it's not related to Matlab plugin itself. Good that you've found a workaround.

Windows line separators may cause tests to fail. It should be fixed by this commit 20efda81ba1640264f2b2ebc616ef669f128576f. Also there is another ticket for wrong line endings #97, so I'll close this one
